results/.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Looks like this file may have been checked in by accident. Let's remove?

It might also be helpful to add .vscode to your gitignore file!

Reply:
done